### DISCUSSION TITLE
How did you scrape the video src links

### DISCUSSION THREAD

**Discussion by yashkathe** [pystardust/ani-cli#852] [Q&A] [answered]
I tried to look at the code and was confused because the video src links are blob links and you cant actually use them right ? So how did you get the original video src links

**Comment by justchokingaround** [comment, marked as answer]
you have to go to inspect tools -> network, and then look through the requests being made, then reverse engineer the process and implement it with curl, i suggest you take a look at https://github.com/Blatzar/scraping-tutorial/ if you're interested to learn more about how it's actually done

**Reply by yashkathe** [reply]
Thanks a lot. this is very helpful
